CMakeLists.txt

```diff
@@ -210,7 +210,7 @@ if(USE_CUDA)
   find_package(CUDA REQUIRED)
   add_definitions(-DMSHADOW_USE_CUDA=1)
   if(FIRST_CUDA AND (NOT USE_OLDCMAKECUDA))
-    if(CUDA_TOOLSET STREQUAL "")
+    if(NOT CUDA_TOOLSET)
```

Comment:
may be ()NOT CUDA_TOOLSET) OR ()CUDA_TOOLSET STREQUAL "") is good

Reply:
NOT checks for empty string and when the variable is not set, so there is no need for the OR

Reply:
ok